README.md

```diff
@@ -125,6 +123,7 @@ Feel free to [open a PR](https://github.com/DenverCoder1/readme-typing-svg/issue
 | `background` |             Background color of the text (default: `00000000`)              | string  |  Hex code without # (eg. `FEFF4C`)  |
 |   `center`   |    `true` to center text or `false` for left aligned (default: `false`)     | boolean |          `true` or `false`          |
 |  `vCenter`   |  `true` to center vertically or `false`(default) to align above the center  | boolean |          `true` or `false`          |
+|    `rtl`     |   `true` to use right-to-left script (Arabic, Hebrew…) (default: `false`)   | boolean |          `true` or `false`          |
```

Comment:
What if instead of having a parameter, it could determine the direction to display the text based on the directionality of the first unidirectional character (eg. if it starts with an English letter it's LTR, if it starts with Arabic it's RTL, standard punctuation is ignored)? Alternatively, it could be determined based on a count comparison of RTL vs LTR characters. By making it automatic, it could simplify configuration and the demo site.

Feel free to let me know what you think.